test/transcoding/registerallocmode.ll

```diff
@@ -6,16 +6,18 @@
 
 ; CHECK-SPIRV: 8 Decorate 4 UserSemantic "num-thread-per-eu 4"
 ; CHECK-SPIRV: 8 Decorate 6 UserSemantic "num-thread-per-eu 8"
-; CHECK-SPIRV-NOT: 8 Decorate 8
+; CHECK-SPIRV  8 Decorate 8 UserSemantic "num-thread-per-eu 0"
```

Comment:
```suggestion
; CHECK-SPIRV:  8 Decorate 8 UserSemantic "num-thread-per-eu 0"
```

Reply:
Also, it seems like I've missed this during review of the initial patch.
Please do not add word count check of the instruction aka "8 Decorate 8" -> "Decorate 8". Also in remaining "Decorate 8" '8' is an ID of something, please explicitly add a check for this something and reuse its ID as a CHECK variable.

Reply:
whoops, all fixed, thanks